aiofcm/connection.py

```diff
@@ -184,9 +183,9 @@ def __init__(
         self.max_connections = max_connections
         self.max_connection_attempts = max_connection_attempts
         self.loop = loop or asyncio.get_event_loop()
-        self.connections = []
+        self.connections: List[FCMXMPPConnection] = []
         # Python 3.10+ does not use the "loop" parameter
-        if sys.hexversion >= 0x030A00F0:
+        if sys.version_info >= (3, 10):
```

Comment:
@Fatal1ty 
i changed this to use `sys.version_info` instead of `sys.hexversion`.
because mypy doesn't understand `sys.hexversion`

https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks